src/components/Calculator.js

```diff
@@ -0,0 +1,60 @@
+import React from 'react';
+import Keys from './keys';
+import calculate from '../logic/calculator';
+
+class Cal extends React.PureComponent {
```

Comment:
Well done for coming this far with the project. 👏 

The projects works as expected, however, the [previous milestone](https://github.com/microverseinc/curriculum-react-redux/blob/main/math-magicians/project_components.md#) required you to convert all class components into functional components and use hooks to replace the class component life-cycle methods. 

I apologise for the confusion because this should have been check in the previous milestone. 🙏

Kindly convert this class component to a functional component and use the useState to replace state and setState to manage data in your store. 💪

Reply:
Hi @brytebee,
thank you for the feedback, however the requirements you are requesting for is in the next milestone as the study on Hooks. Thank you.